contrib/completion/zsh/_docker

```diff
@@ -462,6 +462,8 @@ __docker_subcommand() {
         "($help)*--cap-drop=[Drop Linux capabilities]:capability: "
         "($help)--cidfile=[Write the container ID to the file]:CID file:_files"
         "($help)*--device=[Add a host device to the container]:device:_files"
+        "($help)--device-read-bps=[Limit the read rate (bytes per second) from a device]:device:IO rate: "
+        "($help)--device-write-bps=[Limit the write rate (bytes per second) to a device]:device:IO rate: "
```

Comment:
Just saw @sdurrheimer comment in your other PR (yay, I'm learning!) these should probably have a a `*` in front of `--`, so `"($help)*--device...`